.github/workflows/deploy-previews.yml

```diff
@@ -289,7 +289,7 @@ jobs:
       url: ${{ steps.preview-url.outputs.url }}
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
```

Comment:
[use v3 of GitHub checkout(node-16)](https://github.com/airqo-platform/AirQo-api/pull/1213/commits/07ac9c381956b97bf250f371fd576958931aaa5d)